app/assets/javascripts/spree/backend/spree_product_assembly/index.js.coffee

```diff
@@ -0,0 +1,82 @@
+#= require ./translations
+
+$(document).ready ->
+  Spree.routes.available_admin_product_parts = (productSlug) ->
+    Spree.pathFor("admin/products/" + productSlug + "/parts/available")
+
+  showErrorMessages = (xhr) ->
+    response = JSON.parse(xhr.responseText)
+    show_flash("error", response)
+
+  partsTable = $("#product_parts")
+  searchResultsTable = $("#search_hits")
+
+  search_for_parts = ->
```

Comment:
Yeah, underscore-cased variable names are a hold-over from the original JS — good catch